docs/blog/newsletter-may-2017.rst

```diff
@@ -0,0 +1,111 @@
+
+* With teams that perform regression testing and user acceptance testing (UAT) after a scheduled sprint, docs are sometimes developed during these post-sprint cycles. In this kind of workflow, docs and testing are often closely allied.
+
+* Some writers are embedded on Agile dev teams, attend standups, and work on the regular sprint cycle, with stories assigned for doc; others work with parallel sprints, and devs/PMs serve as stakeholders.
+
+* Some of the "embedded" writers document features at the start of development, but doc is not released until feature complete. Other writers find that features change too much during dev cycles for this approach to be worthwhile.
+
+* Several folks who serve as lone writers wrestle with documenting multiple products or conditional doc sets addressed to different audiences. Some manage by getting developers more involved with helping produce doc content.
+
+-----------------------------
+
+Screen Readers and Accessibility
+################################
+
+A question about dividing a complicated data model into clickable sub-parts brought up an interesting secondary question: can screen readers parse .svg flowcharts?
+
+According to one Slack member, probably not. She explained, screen readers tend to go top-to-bottom, left-to-right, reading all the words and links. They can also read alphabetically by first letter and hierarchical by H1, H2, H3. But more complex stuff (like reading SVG flowcharts) is probably beyond the capability of common screen readers. For complex flowcharts, consider offering a text explanation along with the image.
+
+If you're interested in improving doc accessibility, check out @susanf's `list of accessibility resources <https://docs.google.com/document/d/19crjGz7lryYlvaIEzK8KTX_oMLAxzxx1AwGsjlsymgs/edit#heading=h.c5b1b4c0b5i6>`_. For an alt-text-centered guide, try Allen Flavell's article, `Use of ALT texts in IMGs <http://www.alanflavell.org.uk/alt/alt-text.html>`_.
+
+To address the original question about dividing a large data model into clickable sub-parts, commenters mentioned quite a few SVG tools:
+
+* `SVGOMG <https://jakearchibald.github.io/svgomg>`_
+* `Snapsvg <http://snapsvg.io/>`_
+* `Animatron <https://www.animatron.com/>`_
+* `Inkscape <https://inkscape.org/>`_
+* `Tom Sawyer <https://www.tomsawyer.com>`_
+* `Linkurious <https://linkurio.us>`_
+* `Neo4j <https://neo4j.com/>`_
+* ...and good old `Visio <https://products.office.com/en-us/visio/flowchart-software>`_
+
+------------------------------------------
+
+Peer-to-Peer Teaching
+#####################
+
+When you're tasked with leading a workshop that will help your non-tech-writing colleagues write in clear, plain English, how can you help without embarrassing or alienating anyone? The discussion produced many thoughtful suggestions, including having the group dissect your own writing, but one idea stood out: peer-to-peer teaching using a whiteboard carousel. Here's how it works:
```

Comment:
the current placement of the 'your' at the end of this graf is a little unclear... maybe we could use this instead: 
'...including offering up some of your own writing for the group to dissect'?